src/evaluate-by-operator/operator/divide.js

```diff
@@ -11,9 +11,10 @@ export default function func(first, ...rest) {
   // if (result === Infinity) {
   //   throw Error(ERROR_DIV_ZERO);
   // }
-  if (isNaN(result)) {
-    throw Error(ERROR_VALUE);
-  }
+
+  // if (isNaN(result)) {
```

Comment:
Instead of commenting this out entirely, can we specifically detect the case where we are doing `0/0`?

Reply:
In other words, "foo" / 0 should still throw an error here, but 0 / 0 should return NaN